content/2023-berlin/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Overview
+---
+
+## Overview
+
+PromCon EU 2023 is the eigth conference fully dedicated to the
```

Comment:
```suggestion
PromCon EU 2023 is the eighth conference fully dedicated to the
```

Or just say "8th" :)

Reply:
Thanks! This seems more elevated :smiley: